rct229/ruletest_engine/ruletest_jsons/ashrae9012019/section11/rule_11_6.json

```diff
@@ -0,0 +1,142 @@
+{
+    "rule-11-6-a": {
+        "Section": 11,
+        "Rule": 6,
+        "Test": "a",
+        "test_description": "A one story building, single zone building has service hot water piping. The service hot water piping losses are correctly modeled.",
+        "expected_rule_outcome": "pass",
+        "standard": {
+            "rule_id": "11-6",
+            "ruleset_reference": "Table G3.1 #11, baseline column, i",
+            "rule_description": "Piping losses shall not be modeled.",
+            "applicable_rmr": "Baseline Model",
+            "rule_assertion": "=",
+            "comparison_value": "Expected Value",
+            "mandatory_rule": "No",
+            "schema_version": "0.0.34"
+        },
+        "rmd_transformations": {
+            "baseline": {
+                "id": "ASHRAE229 1",
+                "calendar": {
+                    "is_leap_year": false
+                },
+                "data_timestamp": "2024-02-12T12:00Z",
+                "ruleset_model_descriptions": [
+                    {
+                        "id": "RMD 1",
+                        "buildings": [
+                            {
+                                "id": "Building 1",
+                                "building_segments": [
+                                    {
+                                        "id": "Building Segment 1",
+                                        "zones": [
+                                            {
+                                                "id": "Thermal Zone 1",
+                                                "floor_name": "Floor1",
+                                                "spaces": [
+                                                    {
+                                                        "id": "Space 1",
+                                                        "service_water_heating_uses": [
+                                                            {
+                                                                "id": "SHW 1",
+                                                                "served_by_distribution_system": "SHW Distribution 1"
```

Comment:
I believe the value of ```served_by_distribution_system``` should be the same as the value of the ```id``` under the ```service_water_piping```. Could you check?

Reply:
Hi Yun Joon,

The schema seems to indicate that this should match with a `ServiceWaterHeatingDistributionSystem`

https://github.com/pnnl/ruleset-checking-tool/blob/RT/JG/ruletest_11_6/rct229/schema/ASHRAE229.schema.json#L4765

Let me know if this complicates things on you all's end, but I believe this implementation is correct. Let me know if you all think overwise.

Reply:
Juan, You're right. Thank you for pointing this out. I'll correct the 11-6 rule.